### ISSUE TITLE
Uncaught TypeError: Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by avdg** [imdone/imdone-atom#5]
1. Open the imdone-atom pane
2. Click the list icon with numbers in front to create a new list
3. Give the list a name and press enter
4. Crash

**Atom Version**: 1.0.4
**System**: Microsoft Windows 8.1
**Thrown From**: [imdone-atom](https://github.com/imdone/imdone-atom) package, v1.1.9

### Stack Trace

Uncaught TypeError: Cannot read property 'indexOf' of undefined

```
At C:\Users\Anthony\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:342

TypeError: Cannot read property 'indexOf' of undefined
  at Repository.getFullPath (C:\Users\Anthony\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:342:11)
  at Repository.repo.fileOK (C:\Users\Anthony\.atom\packages\imdone-atom\node_modules\imdone-core\lib\mixins\repo-fs-store.js:80:18)
  at EventEmitter.<anonymous> (C:\Users\Anthony\.atom\packages\imdone-atom\node_modules\imdone-core\lib\mixins\repo-watched-fs-store.js:72:14)
  at emitTwo (events.js:87:13)
  at EventEmitter.emit (events.js:169:7)
  at EventEmitter.FSWatcher.emit (C:\Users\Anthony\.atom\packages\imdone-atom\node_modules\imdone-core\node_modules\chokidar\index.js:365:31)
  at C:\Users\Anthony\.atom\packages\imdone-atom\node_modules\imdone-core\node_modules\chokidar\index.js:281:18
  at FSWatcher.<anonymous> (C:\Users\Anthony\.atom\packages\imdone-atom\node_modules\imdone-core\node_modules\chokidar\index.js:247:7)
  at emitTwo (events.js:87:13)
  at FSWatcher.emit (events.js:169:7)
  at FSEvent.FSWatcher._handle.onchange (fs.js:1215:12)

```

### Commands

```
     -5:16.7.0 settings-view:open (atom-text-editor.editor.is-focused)
  3x -4:30.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -4:24.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:04.6.0 imdone-atom:tasks (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "eslint",
      "atom-lint",
      "lcov-info",
      "sublime-tabs"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-paths, v1.0.2
build, v0.43.0
color-picker, v2.0.11
imdone-atom, v1.1.9
linter, v1.3.2
linter-coffeelint, v1.0.2
linter-jshint, v1.1.4
mocha-test-runner, v0.4.4
travis-ci-status, v0.16.0

# Dev
No dev packages
```

**Comment by piascikj** [comment]
I believe I've found the issue.  It's in imdone-core where it checks if the path is to .imdone/config.json.  Since you're running on windows it needs to be using a backslash \.  I'll get it patched tonight.

**Comment by piascikj** [comment]
Fixed with imdone-core 1.2.2

**Comment by jbjonesjr** [comment]
I'm seeing this issue now, but when trying to open the pane initially. Does it not like the project folders I have included?
